tests/e2e/utils/src/components.js

```diff
@@ -426,6 +426,42 @@ const createCoupon = async ( couponAmount = '5', discountType = 'Fixed cart disc
 	return couponCode;
 };
 
+/**
+ * Adds a shipping zone along with a shipping method.
+ *
+ * @param zoneName Shipping zone name.
+ * @param zoneLocation Shiping zone location. Defaults to United States (US).
+ * @param zipCode Shipping zone zip code. Defaults to empty one space.
+ * @param zoneMethod Shipping method type. Defaults to flat_rate (use also: free_shipping or local_pickup)
+ */
+const addShippingZoneAndMethod = async ( zoneName, zoneLocation = 'United States (US)', zipCode = ' ', zoneMethod = 'flat_rate' ) => {
+	await merchant.openNewShipping();
+	
+	// Fill shipping zone name
+	await page.waitForSelector('input#zone_name');
+	await expect(page).toFill('input#zone_name', zoneName);
+
+	// Select shipping zone location
+	// (.toSelect is not best option here because a lot of &nbsp are present in country/state names)
+	await expect(page).toFill('#zone_locations', zoneLocation);
+	await page.keyboard.press('Tab');
+	await page.keyboard.press('Enter');
+
+	// Fill shipping zone postcode if needed otherwise just put empty space
+	await expect(page).toClick('a.wc-shipping-zone-postcodes-toggle');
+	await expect(page).toFill('#zone_postcodes', zipCode);
+	await expect(page).toMatchElement('#zone_postcodes', zipCode);
+	await expect(page).toClick('button#submit');
+
+	// Add shipping zone method
+	await page.waitFor(1000); // avoiding flakiness
```

Comment:
Are these waits something we could replace with the `await uiUnblocked()`page util we have instead?

Reply:
Unfortunately, no. A simple wait needed here.